readme.md

```diff
@@ -106,7 +114,25 @@ Then, make each route respond with a one-word string containing the RESTful acti
 var express = require("express");
 var app = express();
 
-// Your code starts here...
+// read
+app.get("/songs", function (req,res){
+  res.send("index")
+});
+
+// create
+app.post("/", function (req, res) {
+  res.send("new")
```

Comment:
Post performs "create", the "new" action is a get method that displays the form.